gitlab_stats/utils.py

```diff
@@ -2,7 +2,6 @@
 import datetime
 import os
 
-import gitlab_stats
```

Comment:
I think it is used L87 for the CLI report.

Reply:
Oops, my bad!